database_admin/migrations/025_packages.up.sql

```diff
@@ -0,0 +1,54 @@
+CREATE TABLE IF NOT EXISTS package_name
+(
+    id   INT GENERATED BY DEFAULT AS IDENTITY NOT NULL PRIMARY KEY,
+    name TEXT                                 NOT NULL CHECK ( NOT empty(name)) UNIQUE
+);
+
+GRANT SELECT, INSERT, UPDATE, DELETE ON TABLE package_name TO vmaas_sync;
+GRANT SELECT ON TABLE package_name TO evaluator;
+GRANT SELECT ON TABLE package_name TO listener;
+GRANT SELECT ON TABLE package_name TO manager;
+
+GRANT SELECT, USAGE ON SEQUENCE package_name_id_seq TO evaluator;
+GRANT SELECT, USAGE ON SEQUENCE package_name_id_seq TO listener;
+GRANT SELECT, USAGE ON SEQUENCE package_name_id_seq TO vmaas_sync;
+
+CREATE TABLE IF NOT EXISTS package
+(
+    id          INT GENERATED BY DEFAULT AS IDENTITY NOT NULL PRIMARY KEY,
+    -- TODO: inline vs extract ?
+    name_id     INT                                  NOT NULL REFERENCES package_name,
+    version     TEXT                                 NOT NULL CHECK (NOT empty(version)),
```

Comment:
We can consider a separate table for version too. So we can add TODO here as well.

Reply:
I only extracted name because it is expected to be duplicated, I don't think this is the same for version. In VMaaS we have separate table for EVR because it's duplicated across architectures